src/LiveQuery/ParseLiveQueryServer.js

```diff
@@ -82,6 +89,11 @@ class ParseLiveQueryServer {
         logger.error('unable to parse message', messageStr, e);
         return;
       }
+      if (channel === Parse.applicationId + 'clearCache') {
+        this._clearCachedRoles(message.userId);
```

Comment:
I'm not a huge fan of non-awaited promises. Background jobs not triggered by a user (not like Parse Cloud Jobs) could lead to issues during for example in an integration testing suit of a developer.
I don't know how many Promise are not awaited into parse, but we should definitely avoid this pattern, also an example about a run time issue. On GCP cloud run or serverless solutions, CPU is only allocated during request execution, so background tasks are not ensured and could lead to inconsistency.

The correct pattern, but out of scope of this PR, is to create transactional task queue system.

btw, i think that the cache clean up in your use case is fast try to add a `console.time("speed")` and `console.timeEnd("speed") ` around `this._clearCachedRoles(message.userId);` i think it's okay currently to await this promise.

I'll also recommend in another PR to add the eslint rule to prevent non awaited promise by default like https://eslint.org/docs/rules/require-await

Reply:
Would you like to open another issue?

Reply:
Resolve as out of scope https://github.com/parse-community/parse-server/pull/8026#issuecomment-1152835344